projects/mercury/src/metadata/views/MetadataViewContext.js

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+// import {useHistory} from 'react-router-dom';
```

Comment:
Do we want to deliberately keep these comments in the code?

Reply:
No, thanks for spotting that, those are removed now